source/Calamari/Kubernetes/Commands/HelmUpgradeCommand.cs

```diff
@@ -149,7 +149,7 @@ string PackageDirectory(string packageReferenceName)
             var packageRoot = packageReferenceName;
             if (string.IsNullOrEmpty(packageReferenceName))
             {
-                packageRoot = variables.Get(PackageVariables.IndexedPackageId(packageReferenceName ?? ""));
+                packageRoot = PackageName.ExtractPackageNameFromPathedPackageId(variables.Get(PackageVariables.IndexedPackageId(packageReferenceName ?? "")));
```

Comment:
nit: empty strings are better represented by string.Empty, shows that is was intentionally and empty string

don't fix this now due because Calamari 👎

Reply:
legacy code :)

Reply:
> legacy code :)
it is always legacy code